src/main/java/com/codingnomads/betty/presentation/controller/SearchController.java

```diff
@@ -31,6 +31,7 @@ public SearchController(ProcessTweetsThroughNlpService processTweetsThroughNlpSe
     public String estimateTeamOdds(@RequestParam(name = "teamName", required = false) String teamName, Model model) {
         model.addAttribute("inputTeam", new InputTeam());
         model.addAttribute("teamProbabilityToWin", analyzeTweetsService.calculateProbabilityWithTweets(teamName));
+        twitterService.searchTweetFromAccounts("", "2019-05-10");
```

Comment:
How come this first parameter is hardcoded as empty? Is it because of the hard-coded values in the repository? It might be better to make both of these parameters dynamic (fields or pojos) instead of hardcoding them

Reply:
Yeah I should've left a comment to say this needs to be refactored later. I wrote this so I could test if the search queries work, but eventually spring batch should just automatically trigger the construction of the search queries when there is a new match updated into the db.